x-pack/plugins/security_solution/public/assistant/provider.tsx

```diff
@@ -166,14 +207,13 @@ export const AssistantProvider: FC<PropsWithChildren<unknown>> = ({ children })
       docLinks={{ ELASTIC_WEBSITE_URL, DOC_LINK_VERSION }}
       basePath={basePath}
       basePromptContexts={Object.values(PROMPT_CONTEXTS)}
-      baseQuickPrompts={BASE_SECURITY_QUICK_PROMPTS} // to server and plugin start
-      baseSystemPrompts={BASE_SECURITY_SYSTEM_PROMPTS} // to server and plugin start
       baseConversations={baseConversations}
       getComments={getComments}
       http={http}
       navigateToApp={navigateToApp}
       title={ASSISTANT_TITLE}
       toasts={toasts}
+      currentAppId={currentAppId ?? 'securitySolutionUI'}
```

Comment:
use `APP_UI_ID` from:
```
import { APP_UI_ID } from '../../common';
```

Reply:
In the future PR https://github.com/elastic/kibana/pull/179188/files for global assistant I'm planning to move this provider logic to `elastic_assistant` plugin where I will need the dynamic currentAppId